posts/lte-update.md

```diff
@@ -0,0 +1,131 @@
+we need a library of results about homological complexes, long exact sequences, and such.
+A fundamental input is of course the snake lemma,
+and this was formalized by Riccardo Brasca and Adam Topaz in the fall.
+Andrew Yang is currently working towards a formal proof
+that the homotopy category of complexes is a pretriangulated category.
+This will give us important tools to manipulate the `Ext`-groups
+into a shape that reduces to `first_target`.
+
+## Spaces of measures
+
+As mentioned above, the space `ℳ_{p'} S` is a real condensed vector space
+that is morally a space of measures.
+One of the crucial insights in the proof is that it is a quotient of an arithmetic analogue
+that we've been calling the space of *Laurent measures*: `ℳ(S, ℤ((T))_{r'})`,
+with `(1/2)^{p'} = r'`.
+This quotient map turns out to be the cokernel of an injective endomorphism of the space of Laurent measures.
+Together, these two maps form a short exact sequence,
+which leads to a long exact sequence of `Ext`-groups.
+This quotient map sits in a short exact sequence, together with its kernel.
+Hence the vanishing of the `Ext`-groups above can be reduced to a question about similar `Ext`-groups
+but this time involving these spaces of Laurent measures.
+
+In turn, these spaces of Laurent measures naturally admit a free module as submodule,
+and the quotient is the mysterious space `Mbar r' S` that appears in `first_target`.
+Together, these two short exact sequences are crucial inputs
+that will be combined with the homological tools mentioned above
+to reduce the main theorem to `first_target`.
+
+Filippo A.E. Nuccio has been working arduously on the formalization of these results,
+which amounts to Theorem 6.9 of [Analytic.pdf](https://www.math.uni-bonn.de/people/scholze/Analytic.pdf).
+
+## Breen-Deligne resolutions and MacLane's Q'-construction
+
+Finally, to compute the `Ext`-groups, at some point projective or injective resolutions must enter the picture.
+The proof in [Analytic.pdf](https://www.math.uni-bonn.de/people/scholze/Analytic.pdf)
+relies on so-called Breen-Deligne resolutions, which have the following two crucial properties:
+
+1. It is a functorial construction `A ↦ C(A)` that sends an abelian group (or sheaf) to a complex of abelian groups (or sheaves).
+2. When viewed as a functor to the homotopy category of complexes, it is additive.
+  In other words, `C(A ⊕ A)` is naturally homotopic to `C(A) ⊕ C(A)`.
+
+In addition to these properties,
+Breen-Deligne resolutions have the favourable property that the construction yields a projective resolution `C(A)` for every `A`.
+
+For our formalization project, they also come with the significant downside that the proof of the existence of Breen-Deligne resolutions
+relies on technical results from homotopy theory,
+which haven't been formalized yet.
+
+But to our delight, it turns out that there are constructions that satisfy (1) and (2) above,
+and that are good enough for our purposes.
```

Comment:
```suggestion
But to our delight, it turns out that there are related constructions that satisfy (1) and (2) above which are good enough for our purposes and which can be formalized directly.
```
The previous sentence was strange, because the B-D resolution mentioned above is such a construction to begin with.